extensions/microsoft-authentication/src/keychain.ts

```diff
@@ -85,6 +85,11 @@ export class Keychain {
 			return Promise.resolve(undefined);
 		}
 	}
+	onDidChangePassword(listener: () => void) {
```

Comment:
Not clear to me why this indirection is needed and `vscode.authentication.onDidChangePassword` cannot be used directly.